Extensions/PathfindingBehavior/tests/commonpathfindingruntimebehavior.spec.js

```diff
@@ -0,0 +1,161 @@
+const addObstacle = (runtimeScene) => {
```

Comment:
You may want to move this inside `doTestsCommonpathfindingruntimebehavior` to avoid global variables.